kcidev/subcommands/results.py

```diff
@@ -12,13 +12,14 @@
 
 import click
 import requests
+import yaml
```

Comment:
This package is already using toml, is possible to use toml also for filtering?
I would prefer keep the dependency at minimum for more easy distribution packaging and dependency requirements

Reply:
I don' t really thing toml is a good idea for this kind of filter. YAML is pretty straightforward to write for a our users.